examples/enhance/README.md

```diff
@@ -0,0 +1,44 @@
+# Enhance
+
+This project is a showcase for deploying apps with Begin and [Runme Extension](https://marketplace.visualstudio.com/items?itemName=stateful.runme).
+
+## Prerequisites
+
+This project is based on the HTML-first [Enhance](https://enhance.dev/) framework run. It requires [Node.js](https://nodejs.org/en/download) to be installed on your machine.
+
+### Development
+
+Start the project:
+
+```sh { background=true }
+npm start
+```
+
+Open the project in your browser:
+
+```sh { interactive=false }
+open http://localhost:3333/
+```
+
+This will watch the project directory and restart as necessary.
+
+### Deployment
+
+To deploy this project you need to have `begin` CLI installed on your system. To
+install, please run:
+
+```sh { closeTerminalOnSuccess=false interactive=false }
+curl -sS https://dl.begin.com/install.sh | sh
+```
+
+Once installed successfully, you can log into your Begin account via:
+
+```sh
+begin login
```

Comment:
It fails here since it needs to export the binary to the PATH. Maybe add a step asking to export the variables, for example:

```
export BEGIN_INSTALL="/Users/bencho/.begin"
export PATH="$BEGIN_INSTALL:$PATH"
```

Reply:
Interesting, it added the `$PATH` for me automatically